interpreter/syntax/ast.ml

```diff
@@ -135,6 +138,7 @@ and instr' =
   | Ternary of ternop                 (* ternary numeric operator *)
   | Convert of cvtop                  (* conversion *)
   | ExtractLane of extractop          (* extract lane from v128 value *)
+  | Shift of shiftop                  (* shifts for v128 value *)
```

Comment:
Perhaps rename this to SimdShift for clarity? And symmetrically, ExtractLane -> SimdLane?

Reply:
I'll rename Shift here, and extractlane can go in another PR i have for implementing more of extract lane, thanks!